.github/workflows/build-sonarqube.yml

```diff
@@ -0,0 +1,20 @@
+name: "[Bot] Run sonarqube code scanning'"
+on:
+  push:
+    branches:
+      - develop
```

Comment:
were you able to test it yet?
if not, you can always add a temporary `- support/LIVE-9378-add-sonarqube-integration` after the `- develop`

Reply:
Not yet, good suggestion let me try it.